quickwit/quickwit-doc-mapper/src/default_doc_mapper/mapping_tree.rs

```diff
@@ -54,6 +55,124 @@ pub enum LeafType {
     Text(QuickwitTextOptions),
 }
 
+fn value_to_pretokenized<T: ToString>(val: T) -> PreTokenizedString {
+    let val = val.to_string();
+    PreTokenizedString {
+        text: val.clone(),
+        tokens: vec![Token {
+            offset_from: 0,
+            offset_to: 0,
```

Comment:
this is wrong.

Reply:
i've set the offset_to to 1, is that alright, or should that be `text.len()`?